platform/MCU/STM32F2xx/STM32_USB_Device_Driver/inc/include.mk

```diff
@@ -1,3 +1,6 @@
 
 TARGET_USB_FS_PATH = $(PLATFORM_MCU_PATH)/STM32_USB_Device_Driver
 INCLUDE_DIRS += $(TARGET_USB_FS_PATH)/inc
+
+# bootloader_dct.h is referenced in usbd_flash_if.c
+INCLUDE_DIRS += $(PLATFORM_MODULE_PATH)/../bootloader/src/photon
```

Comment:
It feels somewhat weird to introduce a dependency on the bootloader, so perhaps we should simply add an `extern` declaration of the required function to `usbd_flash_if.c` or move `bootloader_dct.h` into `hal`.

Reply:
This can be fixed by simply adding a forward function declaration.

Reply:
Or we can even move the declaration into `dct.h`, which actually resides in `platform` and abolish the `bootloader_dct.h` header.

Reply:
👍